src/test/java/net/sf/jabref/importer/fileformat/BibtexImporterTest.java

```diff
@@ -0,0 +1,70 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.InputStream;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class BibtexImporterTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private BibtexImporter importer;
+
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+        importer = new BibtexImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormat() throws Exception {
+        try (InputStream stream = BibtexImporter.class.getResourceAsStream("BibtexImporter.example.git.bib")) {
+            assertTrue(importer.isRecognizedFormat(stream));
+        }
+    }
+
+    @Test
+    public void testImportEntries() throws Exception {
+        try (InputStream stream = BibtexImporter.class.getResourceAsStream("BibtexImporter.example.git.bib")) {
+            List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
+
+            assertEquals("Heidelberg", entries.get(0).getField("address"));
```

Comment:
There are multiple calls of  `entries.get(0)`. Replace them by creating a new entry with `BibEntry entry = entries.get(0)`.